videomaker/sass/block-patterns/_404-prompt.scss

```diff
@@ -0,0 +1,7 @@
+.container-404 {
+	padding-top: 170px;
```

Comment:
I'm not that keen on duplicating the padding top values here. I wonder if we can move them inside the header template?

Reply:
If we move it inside the template then child themes would need to also keep their specifics in the template.  There are two examples (Skatepark and Videomaker) where that value is customized.  Keeping them in their template would then require the entire block pattern to be duplicated in the child theme.  For the difference of that gap that level of duplication doesn't seem worthwhile to me.  

If it were implemented as a variable that would eliminate the need for custom CSS but (likely) introduce a custom variable and I'm not keen on using variables in patterns as they don't show the actual value when that pattern is used in the FSE.

Reply:
> I would wager that that's not a small undertaking though;

You may be right. I'll have a look into it. https://github.com/Automattic/themes/pull/5031 has been merged so there's no urgency for this one now.

Reply:
Not for urgency, but to keep the changes focused my 'druthers is to refactor that header gap across the themes in a separate pass.

Reply:
Maybe we can move the group out of the pattern? Since we only need the text, and just add the rest of the changes in the template markup for videomaker?

Reply:
Ah, right @MaggieCabrera that would work and is a simple solution.  I'll make that change.